### ISSUE TITLE
Old data remains in CSV files after clearing logs from the tag

### ISSUE THREAD

**Issue by adanger** [arribada/horizon-gui#91]
Expectation: New CSV files should not contain data from deployments if the tag was cleared of its logs

Actual: CSV files from my tag contain data from previous deployments even after clearing the logs on the tag using the GUI. I have tried clearing the logs using the Erase Log button or run the QuickStart Clear script.

![image](https://user-images.githubusercontent.com/53336446/112022504-fceedd80-8af7-11eb-8170-41915e4bf233.png)
![image](https://user-images.githubusercontent.com/53336446/112022520-011afb00-8af8-11eb-81b1-0b34b720b795.png)

You can see from the following image of my CSV files that even though I cleared the logs and ran the tag on March 17, 2021, there is still GPS data from Aug, Sept, Oct, & Nov of 2020.

![image](https://user-images.githubusercontent.com/53336446/112023161-99b17b00-8af8-11eb-85d0-6ed075fb9bb7.png)

**Note. The JSON and Binary files for March 17, 2021 do NOT contain any data from 2020. Only the CSV files.

**Comment by Flustered** [comment]
Yes, I noticed this.

Fix note:
horizon_gui.py needs to clear /logdownload BEFORE it uses it.  
See: downloadSeparated ():

**Comment by adanger** [comment]
I just tested both the Erase Log and Quick Clear scripts again with v4.0.3 to see if they would erase logs on the tag. It appears they do not, but I don't get any error code in python. Below are two screenshots I took after using Erase Logs or QuickClear that show logs still present on the tag. The python script is next to them.

![image](https://user-images.githubusercontent.com/53336446/113189839-e0018b00-9218-11eb-9ea8-d50ac69d27ff.png)

![image](https://user-images.githubusercontent.com/53336446/113189855-e5f76c00-9218-11eb-917a-62243a94f199.png)
